addons/account/wizard/setup_wizards.py

```diff
@@ -151,3 +152,7 @@ def validate(self):
         extension hook in account_bank_statement_import.
         """
         return self.env["onboarding.onboarding.step"].action_validate_step("account.onboarding_onboarding_step_bank_account")
+
+    def _compute_company_id(self):
+        if not self.company_id:
+            self.company_id = self.env.company
```

Comment:
I know it's stupid but we are able to create several wizard at the same time by code and it should work :man_shrugging: 
```suggestion
        for wizard in self:
            if not wizard.company_id:
                wizard.company_id = self.env.company
```